client/shaders/second_stage/opengl_fragment.glsl

```diff
@@ -68,15 +68,14 @@ vec3 uncharted2Tonemap(vec3 x)
 
 vec4 applyToneMapping(vec4 color)
 {
-	color = vec4(pow(color.rgb, vec3(2.2)), color.a);
 	const float gamma = 1.6;
 	const float exposureBias = 5.5;
 	color.rgb = uncharted2Tonemap(exposureBias * color.rgb);
 	// Precalculated white_scale from
 	//vec3 whiteScale = 1.0 / uncharted2Tonemap(vec3(W));
 	vec3 whiteScale = vec3(1.036015346);
 	color.rgb *= whiteScale;
-	return vec4(pow(color.rgb, vec3(1.0 / gamma)), color.a);
```

Comment:
Why do you no longer need the light-space transformations?

Reply:
They looked very strange to me and created very harsh contrast and oversaturated colors, which as far as I know is exactly what the gamma correction was there to prevent. I’ll post a comparison later.

Reply:
So actually this has to do with #14109, I'll have to look deeper into that...

Reply:
I promised screenshots, here they are. THe PR as is:
![screenshot_20240407_203109](https://github.com/minetest/minetest/assets/72752000/f6d5417e-3d2c-423e-924a-de78365509f6)

With gamma correction before color grading:
![screenshot_20240407_203054](https://github.com/minetest/minetest/assets/72752000/44471bbd-df7f-41ae-8213-e851b0d57754)

I suppose the solution here might be to return everything to how it was and move the color grading step into the tone mapping function?